### ISSUE TITLE
event only gets triggered once

### ISSUE THREAD

**Issue by mnts** [Polymer/lit-element#859]
for example if I have <span @click='${this.handle}'>Button</span> , it gets triggered only once.. more clicks produces no result and I havent seen anywhere in doc where it would be mentioned, that it works only once.. how do I fix that ?

**Comment by web-padawan** [comment]
Please see the possible problem here https://github.com/Polymer/lit-html/issues/1011#issuecomment-535378868

**Comment by sorvell** [comment]
We'll need more information to help. A reproduction would be very useful. Please see https://github.com/Polymer/lit-element/blob/master/CONTRIBUTING.md#filing-bugs.

**Comment by sorvell** [comment]
We're closing this due to lack of response, but please feel free to open a new issue if necessary with a reproduction. Please see https://github.com/Polymer/lit-element/blob/master/CONTRIBUTING.md#filing-bugs.
